src/builder/build_traits.rs

```diff
@@ -604,7 +604,7 @@ fn wire_up<T: Dataflow + ?Sized>(
         }
     }
 
-    if let Some((connected, connected_offset)) = base.hugr().linked_port(src, src_offset) {
+    if let Some((connected, connected_offset)) = base.hugr().linked_ports(src, src_offset).next() {
```

Comment:
Should we be iterating over _all_ linked ports here?

Reply:
This is for the explicit-copies builder. This part of the code is directly dropped in the next PR #140